### PR TITLE
G2P data download page

### PR DESCRIPTION
**Pull request by seeta-ramaraju**
Changes related to Jira task [G2P-166](https://www.ebi.ac.uk/panda/jira/browse/G2P-166)

### CODE REVIEW

File: src/views/DataDownloadView.vue [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+      v-if="dataDownloadErrorMsg"
+    >
+      <div>
+        <i class="bi bi-exclamation-circle-fill"></i> {{ dataDownloadErrorMsg }}
+      </div>
+    </div>
+    <div v-if="!isDataLoading">
+      <p>
+        Download format is available
+        <a href="" target="_blank" style="text-decoration: none">here</a>.
```
Comment (dglemos) [inline src/views/DataDownloadView.vue:148]: We should have a README file under www.ebi.ac.uk/gene2phenotype/README (localhost:5173/README) and point to that.
I can create the file and send it to you.

File: src/views/DataDownloadView.vue [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+          this.dataDownloadErrorMsg = error.message;
+          console.log(error);
+        });
+    },
+  },
+};
+</script>
+<template>
+  <div class="container px-5 py-3" style="min-height: 60vh">
+    <h2 class="pb-2">Download G2P data by panel</h2>
```
Comment (dglemos) [inline src/views/DataDownloadView.vue:99]: On the main page we call it `Disorder panel`.
This page could be renamed to `Download G2P data by disorder panel`.

File: src/views/DataDownloadView.vue (hunk 1 above)
Comment (seeta-ramaraju) [reply src/views/DataDownloadView.vue:148]: Plan is to host the README file in the FTP site and add a link to it.
I have to add a few files (this README file and other files related to the variant filtering documentation page) on the FTP site, once I've done that, I will raise a separate PR to update the links.

File: src/views/DataDownloadView.vue (hunk 2 above)
Comment (seeta-ramaraju) [reply src/views/DataDownloadView.vue]: taken care in latest commit

**Review by dglemos — APPROVED** [review]

**Pull request merged**